projects/fsastorefrontlib/src/cms-components/user-profile/user-profile.component.html

```diff
@@ -27,8 +27,11 @@ <h5 class="text-center py-3 mb-0">
               : ('dashboard.yourProfile' | cxTranslate)
           }}
         </h5>
-        <div class="user-profile-wrapper background-color-4">
-          <div class="user-profile bg-white border border-color-3 text-center">
+        <div class="user-profile-wrapper background-color-4 overflow-hidden">
+          <div
+            class="user-profile bg-white border border-color-3 text-center slide-out"
```

Comment:
fixing the `slide-out` class like this causes this animation on reload as well. We can either add this class programmatically when needed or create separate task for that. It looks fine, but we should definitely fix it, so it can be perhaps used elsewhere as well.

Reply:
I could use Renderer for this?

Reply:
Yes, for sure. Renderer should (must 😄 ) be used whenever we want to change something in the DOM. It can also be achieved with ElementRef, but then virtual DOM and actual DOM wouldn't be in-sync.